src/packed-pack.ts

```diff
@@ -331,3 +410,11 @@ export async function readPackFile(
         reverseIndex
     );
 }
+
+function isRefDeltaPrelude(obj: ObjectPrelude): obj is RefDeltaPrelude {
```

Comment:
I doubt I get the idea for using such a simple functions. If that's for make TS happy, there is should another solution.